tests/modules/test_dielectricplanar.py

```diff
@@ -164,7 +164,7 @@ def test_multiple_dipole_orientations(self, selection, orientation, M_par, M_per
         For example if some positions of charges are erroneously shifted out of the
         system.
 
-        See https://gitlab.com/maicos-devel/maicos/-/issues/83 for more infos.
+        See https://github.com/maicos-devel/maicos/issues/73 for more infos.
```

Comment:
I just looked at the issue, there is no Info there. No Idea why we linked this. We might as well remove the link.

```suggestion
```